controllers/suite_test.go

```diff
@@ -89,8 +89,9 @@ var _ = BeforeSuite(func() {
 		Expect(err).To(Succeed())
 	}
 
+	uncachedObjects := []client.Object{&batchv1.Job{}}
```

Comment:
Maybe add a comment why we are doing this? If this fixes the issue, we should/must follow this up further...

Reply:
added a fixme